ql/src/java/org/apache/hadoop/hive/ql/exec/JoinUtil.java

```diff
@@ -350,6 +351,7 @@ public static AbstractSerDe getSpillSerDe(byte alias, TableDesc[] spillTableDesc
           + Utilities.ctrlaCode,
           org.apache.hadoop.hive.serde.serdeConstants.LIST_COLUMNS, colNames
           .toString(),
+              hive_metastoreConstants.TABLE_BUCKETING_VERSION, "-1",
```

Comment:
nit. indentation

Reply:
I've removed the usage of this makeProperties...I think it's also more readable this way...